src/api/article/localized/controller.js

```diff
@@ -0,0 +1,31 @@
+import { checkIsFound } from 'utils/validation';
+import { sendJson } from 'utils/api';
+
+import Article from 'api/article/article.model';
+
+import LocalizedArticle from './model';
+
+export const create = async ({ params: { articleId }, body }, res, next) =>
```

Comment:
Redundant `async` cause you don't use `await` inside. Please check other methods too.